stable/datacube-dashboard/values.yaml

```diff
@@ -11,66 +11,121 @@ image:
+  cron: "0 */6 * * *"  ## Run every 6 hours
   suspend: false
   dockerArgs: [ "cubedash-gen", "--all", "||", "true"]
+dockerArgs: [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "4", "--timeout", "60", "cubedash:app" ]
 
-service:
-  type: NodePort
-  port: 80
+update_creation_dt:
+  enabled: false
+  historyLimit: 1
+  cron: "0 13 * * *"  ## Run daily at 11 pm AEST
+  suspend: false
+  # dataset_type_ref id for the desired product is obtained from `SELECT id,name FROM agdc.dataset_type;` psql command
+  dockerArgs:
+    - [ "psql", "-c", "'drop schema if exists cubedash cascade'"]
+    - [ "psql", "-c", "'UPDATE agdc.dataset d SET metadata = jsonb_build_object('creation_dt', s.metadata#>>'{extent, center_dt}') || s.metadata FROM agdc.dataset s WHERE d.dataset_type_ref=81 AND d.id = s.id'"]
+    - [ "psql", "-c", "'UPDATE agdc.dataset d SET metadata = jsonb_build_object('creation_dt', s.metadata#>>'{extent, center_dt}') || s.metadata FROM agdc.dataset s WHERE d.dataset_type_ref=82 AND d.id = s.id'"]
+    - [ "psql", "-c", "'UPDATE agdc.dataset d SET metadata = jsonb_build_object('creation_dt', s.metadata#>>'{extent, center_dt}') || s.metadata FROM agdc.dataset s WHERE d.dataset_type_ref=83 AND d.id = s.id'"]
+    - [ "psql", "-c", "'UPDATE agdc.dataset d SET metadata = jsonb_build_object('creation_dt', s.metadata#>>'{extent, center_dt}') || s.metadata FROM agdc.dataset s WHERE d.dataset_type_ref=84 AND d.id = s.id'"]
 
-setup:
-  dockerArgs: [ "/bin/bash", "-c", "PGPASSWORD=$ADMIN_PASSWORD psql -h $DB_HOSTNAME -p 5432 -U $ADMIN_USERNAME -d $DB_DATABASE -c 'CREATE EXTENSION IF NOT EXISTS postgis'" ]
-  image:
-    registry: docker.io
-    repository: postgres
-    tag: latest
-    pullPolicy: IfNotPresent
+additionalSettings: |
+  CUBEDASH_THEME='dea'
+  CUBEDASH_DEFAULT_PRODUCTS = ('aster_false_colour','ls5_fc_albers','ls7_fc_albers',)
 
-containerPort: 8080 # Internal port on which pod is running and serving cubedash app
+  # Which field should we use when grouping products in the top menu?
+  CUBEDASH_PRODUCT_GROUP_BY_FIELD = 'product_type'
+
+  # Ungrouped products will be grouped together in this size.
+  CUBEDASH_PRODUCT_GROUP_SIZE = 5
+
+  # Maximum search results
+  CUBEDASH_HARD_SEARCH_LIMIT = 100
+
+  # Maximum number of source/derived datasets to show
+  CUBEDASH_PROVENANCE_DISPLAY_LIMIT = 30
+
+  # Customise '/stac' endpoint information
+  STAC_ENDPOINT_ID = 'AWS_Dev_EKS_Explorer'
+  STAC_ENDPOINT_TITLE = 'AWS dev eks explorer'
+  STAC_DEFAULT_PAGE_SIZE = 20
+  STAC_PAGE_SIZE_LIMIT = 1000
 
 ingress:
-  ## Enable Ingress.
-  ##
-  enabled: false
-  host:
+  enabled: true
+  path: "/*"
+  hosts:
+    - "explorer.dev.dea.ga.gov.au"
+  annotations:
+    kubernetes.io/ingress.class: alb
 
-## Define a TCP liveness probe
-## For a liveness probe, giving up means the pod will be restarted.
-livenessProbe:
-  httpGet:
-    path: "/"
-    port: 8080
-  initialDelaySeconds: 10
-  periodSeconds: 10
+    # set ALB parameters
```

Comment:
Same here, need not have to set defaults for ingress. You can handle in each cluster service config

Reply:
Deleted defaults for ingress